lib/feature/add_seed_to_app/enter_seed_phrase/view/enter_seed_phrase_view.dart

```diff
@@ -0,0 +1,414 @@
+import 'dart:async';
+
+import 'package:app/feature/add_seed_to_app/constants.dart';
+import 'package:app/l10n/l10n.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:nekoton_repository/nekoton_repository.dart';
```

Comment:
Whaaat? :)

Why UI knows about repository? We shouldn't use it directly -- only via bloc!

Reply:
-